cmake/developer_package/add_ie_target.cmake

```diff
@@ -72,7 +72,7 @@ function(addIeTarget)
     endforeach()
 
     file(GLOB_RECURSE includes ${includeSearch})
-    file(GLOB_RECURSE sources  ${sourceSearch})
+    file(GLOB_RECURSE sources CONFIGURE_DEPENDS ${sourceSearch})
```

Comment:
@serhii-pavlovskyi-altran, could you please revert these changes in CMake? As we discussed with @ilya-lavrenov, the advantages of this flag and how it will affect configuration time aren't clear for now

Reply:
the advage is you don't have to remember to run cmake after you've added source file and you don't have to run cmake after every branch change in case branches have different sets of files. the disadvantage is it will check globs on every build. on my system it takes about 40 milliseconds now. you can test by running `cmake -P CMakeFiles/VerifyGlobs.cmake` from build directory on this branch. it will take more time if more globs are converted

Reply:
dropped